src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl.xaml

```diff
@@ -118,7 +121,16 @@
                 </Style>
             </TextBox.Style>
         </TextBox>
-        <TreeView Grid.Row="2" Name="SymbolTree">
+        <ProgressBar
+            Name="ProgressBar"
+            Grid.Row="2"
+            Minimum="0"
+            Maximum="100"
+            Height="2"
+            IsIndeterminate="True"
```

Comment:
There's a WPF bug that we can't use just true here :( 

You'll need to make this 

```xml
IsIndeterminate="{Binding RelativeSource={RelativeSource Self}, Path=IsVisible}"
```

#61186 has details on why we have to do this

Reply:
Thanks for bringing this up! I've updated it now

Reply:
@emilyanas2323 Should this still be `="True"`?